extensions/wikia/AdEngine/js/video/player/playerTracker.js

```diff
@@ -64,7 +64,8 @@ define('ext.wikia.adEngine.video.player.playerTracker', [
 				'additional_2': floatingState,
 				'vast_id': params.vastId || emptyValue.string,
 				'video_id': params.videoId || '',
-				'btl': billTheLizard && billTheLizard.hasResponse() ? 1 : 0
+				'btl': billTheLizard && billTheLizard.hasResponse() ? 1 : 0,
+				'user_block_autoplay': params.withAudio
```

Comment:
We should send 1/0 instead of true/false. Look at lines L59-L60. It's already sent to DW. 

However - imo this solution doesn't cover the problem from ticket. We need to know whether ctp was triggered by user (by setting the cookie). Queen_of_hearts/ab test impacts `params.withAudio` so we can't rely on this flag to address this issue.